pkg/defaults/defaults.go

```diff
@@ -307,7 +309,26 @@ func FromConfig(
 		if err != nil {
 			return nil, nil, fmt.Errorf("could not determine promotion defaults: %w", err)
 		}
-		postSteps = append(postSteps, release.PromotionStep(*cfg, config.Images, requiredNames, imageClient, imageClient, jobSpec))
+
+		dstImageClient := imageClient
+
+		if promoteKubeConfigPath != "" {
+			kubeConfigs, _, err := util.LoadKubeConfigs(promoteKubeConfigPath)
+			if err != nil {
+				return nil, nil, fmt.Errorf("failed to load kubeconfigs: %w", err)
+			}
+			// Will be changed to app.ci after migration
+			promoteClusterContext := "api.ci"
```

Comment:
I wouldn't expect there to be more than one context, we can just swap the kubeconfig when we want to change this

Reply:
Done.